cfpb/consumerfinance.gov
# Financial Well-Being: Move js from apps to routes js

Financial Well-Being app has its initialization code inside the routes js directory, which calls js in the apps directory. The apps directory is really for applications that have dedicated templates that call the scripts from apps directly. This doesn't happen with Financial Well-Being.

## Changes

- Move js from apps to routes js


## How to test this PR

1. Pull branch and `gulp clean && gulp build` and http://localhost:8000/consumer-tools/financial-well-being/ and http://localhost:8000/consumer-tools/financial-well-being/results/ should work like production.
